lib/codegen/src/lib.rs

```diff
@@ -63,6 +63,8 @@ extern crate target_lexicon;
 #[macro_use]
 extern crate log;
 
+extern crate rustc_apfloat;
```

Comment:
`rustc_apfloat` uses the `u128` type, which is not stable on rust 1.25.0, which cranelift attempts to support.